ea_airflow_util/dags/aws_param_store_to_airflow_dag.py

```diff
@@ -55,65 +63,40 @@ def insert_all_aws_params_to_airflow():
     # stackoverflow link:
     # https://stackoverflow.com/questions/51863881/is-there-a-way-to-create-modify-connections-through-airflow-api
     @staticmethod
-    def create_conn(
-            conn_id  : str,
-            conn_type: str,
-            host     : str,
-            schema   : str,
-            login    : str,
-            password : str,
-            port     : str,
-            extra    : str
-    ) -> Connection:
+    def create_conn(conn_id: str, **kwargs) -> Connection:
         """
         Store a new connection in Airflow Meta DB
-        TODO: Consider using **kwargs to make this more flexible, if possible.
 
         :param conn_id:
-        :param conn_type:
-        :param host:
-        :param schema:
-        :param login:
-        :param password:
-        :param port:
-        :param extra:
+        :param kwargs:
         :return:
-        """
-        conn = Connection(
-            conn_id=conn_id,
-            conn_type=conn_type,
-            host=host,
-            schema=schema,
-            login=login,
-            password=password,
-            port=port,
-            extra=extra
-        )
 
+        :Keyword Arguments:
+            * conn_type
+            * host
+            * schema
+            * login
+            * password
+            * port
+            * extra
+        """
+        conn = Connection(conn_id=conn_id, **kwargs)
         session = airflow.settings.Session()
-        conn_name = (
-            session
-                .query(Connection)
-                .filter(Connection.conn_id == conn.conn_id)
-                .first()
-        )
-
-        if str(conn_name) == str(conn.conn_id):
+
+        # Verify whether the connection already exists in Airflow.
+        if session.query(Connection).filter(Connection.conn_id == conn.conn_id).first():
             logging.warning(
                 f"Connection {conn.conn_id} already exists!"
             )
             return None
 
-        session.add(conn)
-        session.commit()
-
-        logging.info(Connection.log_info(conn))
-        logging.info(
-            f"Connection {conn_id} was added."
-        )
-
-        return conn
+        else:
+            session.add(conn)
+            session.commit()
 
+            logging.info(Connection.log_info(conn))
+            logging.info(
+                f"Connection {conn_id} was added."
+            )
 
-    def globalize(self) -> None:
```

Comment:
`globalize()` is not actually usable. Globals defined within a function are scoped within that function, not actually globally. This method has been removed in this PR.